tools/integration/test/testConfig.js

```diff
@@ -29,19 +29,25 @@ const components = [
   'go/golang/rsc.io/quote/v1.3.0',
   'nuget/nuget/-/NuGet.Protocol/6.7.1',
   'composer/packagist/symfony/polyfill-mbstring/v1.28.0',
-  // 'pod/cocoapods/-/SoftButton/0.1.0', // Dev and prod have different file counts. See https://github.com/clearlydefined/crawler/issues/529
+  'pod/cocoapods/-/SoftButton/0.1.0', // Dev and prod have different file counts. See https://github.com/clearlydefined/crawler/issues/529
   'deb/debian/-/mini-httpd/1.30-0.2_arm64',
   'debsrc/debian/-/mini-httpd/1.30-0.2'
   // 'sourcearchive/mavencentral/org.apache.httpcomponents/httpcore/4.1' // Dev and prod have different license and scores. See https://github.com/clearlydefined/crawler/issues/533
 ]
 
+//When production response is not available or needs to be corrected, stub response from production service for testing
+const expectedResponses = [
+  { url: '/definitions/pod/cocoapods/-/SoftButton/0.1.0', response: require('./fixtures/softbutton-0.1.0.json') }
+]
```

Comment:
I had an idea that we could just iterate over all files in `./fixtures/` and automagically ✨ determine the coordinates from the contents but maybe this would be too much magic and just end up being confusing in the end.
Wanted to mention it as a thought but definitely good to keep this as is 👍🏼

Reply:
Yes.  Using certain structure in fixture (to capture url and response) can be way to autoload.  Another option is to organize the fixtures by end points, e.g. definition responses for coordinates are grouped under folder /definitions, fixtures can be autoloaded based on end points and coordinates.  If we have more and more fixtures, this can be worthwhile to look into.